v2/internal/reconcilers/arm/arm_client_cache_test.go

```diff
@@ -141,7 +141,7 @@ func Test_ARMClientCache_ReturnsPerResourceScopedClientOverNamespacedClient(t *t
 	g.Expect(err).ToNot(HaveOccurred())
 
 	rg := newResourceGroup("test-namespace")
-	rg.Annotations = map[string]string{reconcilers.PerResourceSecretAnnotation: perResourceCredentialName.Name}
+	rg.Annotations = map[string]string{common.PerResourceSecretAnnotation: perResourceCredentialName.Name}
```

Comment:
I'd prefer if we spread these literal maps across multiple lines, as we already typically do elsewhere.

```suggestion
	rg.Annotations = map[string]string{
		common.PerResourceSecretAnnotation: perResourceCredentialName.Name,
	}
```